csharp/src/Ice/Proxy.cs

```diff
@@ -1583,14 +1583,7 @@ public override void handleInvokeResponse(bool ok, OutgoingAsyncBase og)
     private void iceI_ice_getConnection(OutgoingAsyncCompletionCallback completed, bool synchronous)
     {
         var outgoing = new ProxyGetConnection(this, completed);
-        try
-        {
-            outgoing.invoke(_ice_getConnection_name, synchronous);
-        }
-        catch (Ice.Exception ex)
-        {
-            outgoing.abort(ex);
```

Comment:
The C# code was overall correct: we were catching and handling the exception with abort.

I just move this catch/handle to the invoke method implementation, for consistency with other invoke implementations and other language mappings.